mne/io/eeglab/eeglab.py

```diff
@@ -258,6 +286,24 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
 
         last_samps = [eeg.pnts - 1]
         info = _get_info(eeg, montage, eog=eog)
+
+        n_chan = len(info["chs"])
+        stim_chan = dict(ch_name='STI 014', coil_type=FIFF.FIFFV_COIL_NONE,
+                         kind=FIFF.FIFFV_STIM_CH, logno=n_chan + 1,
+                         scanno=n_chan + 1, cal=1., range=1., loc=np.zeros(12),
+                         unit=FIFF.FIFF_UNIT_NONE, unit_mul=0.,
+                         coord_frame=FIFF.FIFFV_COORD_HEAD)
```

Comment:
This code must be repeated in a bunch of different places. We should probably make it a `_create_stim_ch()` in `mne/io` somewhere so all the non-FIF readers that create events can use it.

Also I doubt the coord_frame is HEAD (it's probably UNKNOWN), can you check what the values are for the Neuromag FIF file in the sample dataset?